### DISCUSSION TITLE
The current campus experts have already graduated!

### DISCUSSION THREAD

**Discussion by AchalVenkatesh** [community/community#60966] [GitHub Education]
### Hi everyone,

Hello everyone!
In the GitHub campus experts portal, our old GitHub campus experts are still holding the position, even though they have already graduated. This is causing us some issues while applying, as it says that our campus already has an expert! 2 experts to be precise. While they have already graduated! Please help us with how to show GitHub that they are alumni now and help us apply to the Campus Experts program! It would help us a lot if we get one campus expert among us, as we want to take our Campus community to greater heights and make it a place for tech enthusiasts to gather and a place for beginners to find guidance with their tech careers.

Although I've been using GitHub for quite some time now, I didn't know about the existence of such a vibrant community!!
Thank you for your time and help!!

**Comment by toukirkhan** [comment]
agreee
